paymentsheet/src/main/java/com/stripe/android/paymentsheet/DisplayableSavedPaymentMethod.kt

```diff
@@ -14,4 +15,21 @@ internal data class DisplayableSavedPaymentMethod(
 
         return isCbcEligible && hasMultipleNetworks
     }
+
+    fun getDescription(resources: Resources) = when (paymentMethod.type) {
```

Comment:
Is there a way to model this that ensures we don't miss this when adding another type to supported saved payment methods?

We have a thing (`PaymentMethodDefinition.supportedAsSavedPaymentMethod`) which might be useful here.

Reply:
I see this isn't new. So maybe just a follow up task?

Reply:
yeah, would be a good improvement! https://jira.corp.stripe.com/browse/MOBILESDK-2223